index.html

```diff
@@ -611,9 +662,9 @@ <h3>Terminology</h3>
         <h3>Data Model</h3>
 
         <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] SHOULD
+A [=controller document=] specifies one or more relationships between
+an [=identifier=] and a set of verification methods and service 
```

Comment:
```suggestion
an [=identifier=] and a set of verification methods and/or service 
```

Reply:
@dlongley — I think you are correct, though I then wonder what a controller document that specifies zero such relationships does, and how then to describe it other than to say "a controller document is a document"?

Reply:
>  how then to describe it other than to say "a controller document is a document"?

Well, I don't think we have to answer or focus on that question, but can rather just say something like "A controller document enables the specification of relationships between an [=identifier=] and/or a set of verification methods...", without saying "X or more" at all.

Reply:
@dlongley That should probably be corrected (in a different PR).

A controller document without a verification method or service endpoint is, IMO, not usable for verifying proofs nor interacting with the controller of an identifier, so I would argue it should NOT be considered a valid DID Document, regardless of the current "zero or more" allowances for both.

Reply:
@jandrieu,

I think it would be better to just simplify the language and not have to debate that. That debate could go in a number of different directions and even include a decision that codifies a mistake where we ban some expression that turns out would have been useful in some way we didn't think of in the moment. So I don't think there's a lot of upside in such a debate or the potential ban resulting from it -- and a minor language tweak here can probably keep us moving forward.

Reply:
@dlongley Agreed. I'll update that in my full pass later that incorporates Ted's request for `<p>` tags.

However, I stand by my point. A DID document without methods or services is a crime against nature.